dashboard/app/models/script.rb

```diff
@@ -2106,4 +2106,10 @@ def get_unit_resources_pdf_url
       Services::CurriculumPdfs.get_unit_resources_url(self)
     end
   end
+
+  # To help teachers have more control over the pacing of certain scripts, we
+  # send students on the last level of a lesson to the unit overview page.
+  def should_show_unit_overview_between_lessons
+    csd? || csp? || csa?
```

Comment:
Hopefully setting it up like this makes it easy to swap in different logic as the requirements evolve or change in the future. I wanted to err on being super specific about why I'm grouping these rather than something like `middle_high_courses`

Reply:
Tiny: could add `?` to the end to signify it returns a boolean and rename `show_unit_overview_between_lessons`